parquet-encoding/src/main/java/org/apache/parquet/bytes/LittleEndianDataOutputStream.java

```diff
@@ -210,4 +212,12 @@ public final void writeDouble(double v) throws IOException {
     writeLong(Double.doubleToLongBits(v));
   }
 
+  public void close() {
+    try {
+      out.close();
+    }catch(IOException e){
+      throw new OutputStreamCloseException(e);
+    }
+  }
```

Comment:
Making this throw IOException seems to require adding a bunch of new throw statements around the codebase, I'll write a helper for this instead.